File: src/wi_childcare_api/src/api/models/childcare_provider.rs
use serde::{Deserialize, Serialize};

#[derive(Serialize, Deserialize, Debug)]
pub struct ChildcareProvider {
    #[serde(rename(deserialize = "UNIQUE_ID"))]
    id: i32,
    #[serde(rename(deserialize = "FACILITY_NAME"))]
    facility_name: String,
    #[serde(rename(deserialize = "ADDRESS"))]
    address: String,
    #[serde(rename(deserialize = "ADDRESS_2"))]
    address_two: String,
    #[serde(rename(deserialize = "CITY"))]
    city: String,
    #[serde(rename(deserialize = "STATE"))]
    state: String,
    #[serde(rename(deserialize = "COUNTY"))]
    county: String,
    #[serde(rename(deserialize = "LAT"))]
    lat: f64,
    #[serde(rename(deserialize = "LON"))]
    long: f64,
    #[serde(rename(deserialize = "CONTACT_PHONE"))]
    provider_phone: String,
    #[serde(rename(deserialize = "APPLICATION_TYPE"))]
    application_type: String,
    #[serde(rename(deserialize = "CAPACITY"))]
    capacity: i16,
}

File: src/wisconsin-dhs-dotnet/WisconsinDhs.Models/ChildcareProvider.cs
namespace WisconsinDhs.Models;

public class ChildcareProvider
{
    public int Id { get; set; }
    public string FacilityName { get; set; }
    public string Address { get; set; }
    public string AddressTwo { get; set; }
    public string City { get; set; }
    public string State { get; set; }
    public string County { get; set; }
    public float  Latitude { get; set; }
    public float Longitude { get; set; }
    public string ProviderPhone { get; set; }
    public string ApplicationType { get; set; }
    public short Capacity { get; set; }
    public string ContactName { get; set; }
}